Automattic/wc-calypso-bridge
# Handling footer credits for Woo Express plans.

### Changes proposed in this Pull Request:


- We'd like to show a specific footer credit for Woo Express sites instead of the default "Blog with" for WordPress.com
- This is the Martech request: https://martechp2.wordpress.com/2023/03/10/add-footer-link-in-ecommerce-plan-sites/
- We first opened a PR in wpcomsh, but were adviced to contain this Woo Express specific code in this repo from @xristos3490 :) 







**Setup**
- If you haven't done this already, do an `npm i && npm run build`
- Make a free site, and tag it as a WoA test site in MC.

- Set up a Woo Express site, either a trial, or one of the paid plans via the SA.
- Checkout this branch
- Transfer all files (I excluded node_modules) to your test site via:
```sh
rsync -avze ssh --delete ./ {{my_woo_test_site}}.wordpress.com@sftp.wp.com:/{{absolute_path_to_htdocs}}/wp-content/mu-plugins/wpcomsh/vendor/automattic/wc-calypso-bridge/
```

... or probably you have a better way to sync. :) 

**Testing**
- Open another terminal window, or Tmux, and ssh into your site. Tail the error log (e.g. `less +F /tmp/php-errors`)
- When reloading your site, you should see that the footer credit is updated to
```html
<div class="wp-block-group"><p class="has-text-align-center">Powered by <a href="https://woocommerce.com/express/?utm_source=referral&amp;utm_medium=footer-credit&amp;utm_campaign=woo-express-footer-credit" rel="nofollow" class="customize-unpreviewable">Woo</a></p></div>
```
- Verify no related PHP errors.

### Other information:




### FOR PR REVIEWER ONLY:
